api/src/main/java/org/openmrs/module/fhir2/api/FhirPatientService.java

```diff
@@ -44,4 +44,6 @@ IBundleProvider searchForPatients(StringAndListParam name, StringAndListParam gi
 	        HashSet<Include> revIncludes);
 	
 	IBundleProvider getPatientEverything(TokenParam identifier);
+	
+	IBundleProvider getPatientEverythingType();
```

Comment:
We could have overloaded methods.
```suggestion
	IBundleProvider getPatientEverything();
```